### PR TITLE
Rework sidenav / page titles to better match

### PR DESCRIPTION
**Pull request by lnhsingh**
Closes DOC-8393

### CODE REVIEW

**Review by nickvigilante — CHANGES_REQUESTED** [review]
One blocking change, one optional nit, but overall LGTM! Let me know if you need help with the blocking change.

File: src/current/_includes/v23.1/sidebar-data/sql.json [hunk 1]
```diff
@@ -974,43 +974,43 @@
                 ]
               },
               {
-                "title": "POINT",
+                "title": "`POINT`",
```
[the remark this answers is not in the document]
Comment (lnhsingh) [reply src/current/_includes/v23.1/sidebar-data/sql.json:977]: Done - Thank you!

File: src/current/_includes/v23.1/sidebar-data/cloud-deployments.json [hunk 2]
```diff
@@ -374,7 +374,7 @@
                     ]
                   },
                   {
-                    "title": "Export Metrics From a {{ site.data.products.dedicated }} Cluster",
+                    "title": "Export Metrics From a Dedicated Cluster",
```
[the remark this answers is not in the document]
Comment (lnhsingh) [reply src/current/_includes/v23.1/sidebar-data/cloud-deployments.json:377]: I think I'll leave this hardcoded for now to make sure the sidenav stays the way we want it to

**Review by nickvigilante — APPROVED** [review]
Beautiful! LGTM!

**Comment by taroface** [comment]
image

Do we want to make this "Overview" for consistency with the naming/renaming elsewhere?

**Comment by lnhsingh** [comment]
@taroface - I think the Security-related edits you proposed will be updated with @MichaelTrestman's [PR](https://github.com/cockroachdb/docs/pull/17595)

**Review by taroface — COMMENTED** [review]
Mostly non-blocking nits/comments except for one slightly inaccurate title and one typo.

**Comment by taroface** [comment]
image

Should "From" be lowercase here?

File: src/current/v23.1/inverted-indexes.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Generalized Inverted Indexes
+title: Index JSON and Array Data with Generalized Inverted Indexes
```
[the remark this answers is not in the document]
Comment (lnhsingh) [reply src/current/v23.1/inverted-indexes.md]: I'll just revert back for now.

**Pull request merged**